### DISCUSSION TITLE
how to get camera position?

### DISCUSSION THREAD

**Discussion by Intervel** [skypjack/entt#1076] [Q&A]
this is my scene :

![3](https://github.com/skypjack/entt/assets/86478025/742a77d1-4a4f-47cf-a201-af25cf62f9ec)

this is what inside render system for newEntity2 :

![2](https://github.com/skypjack/entt/assets/86478025/bb5c2107-b893-4d2f-8032-d12d17863150)

i want to get the cameraEntity. how to do that? is this possible with entt?

![1](https://github.com/skypjack/entt/assets/86478025/6c5d46bc-ee4d-48e3-b4fe-65233fe4e4cd)

i also create a camera3DSystem for newEntity. this system handling movement. it works by changing the transform3DComponent position values.

**Comment by skypjack** [comment]
Since you're using a tag and you have a single camera from what I get, you can do something like this:
```cpp
auto entity = registry.view<Camera3DComponent>().front();`
```
However, I wouldn't create a whole storage for a single element. Instead, you can store `newEntity2` aside in you scene or in the registry context for later uses. It seems straightforward in any case.

**Comment by Intervel** [comment]
@skypjack thank you sir!
